src/protoc_gen_clojure/code_gen_request/transform.clj

```diff
@@ -9,7 +9,7 @@
             [protoc-gen-clojure.specs.msg :as msgdef]
             [protoc-gen-clojure.specs.rpc :as rpcdef]
             [protoc-gen-clojure.ast :as ast]
-            [protoc-gen-clojure.code-gen-request.one-of-transform :refer [adjust-msg-for-oneof]]
+            [protoc-gen-clojure.code-gen-request.one-of-transform :as oneof]
```

Comment:
+1 on this change, the separation into its own namespace is helpful with the name changes above.